src/lib/types.ts

```diff
@@ -158,18 +160,53 @@ export interface v3ProjectsAttributes {
   type: string;
 }
 
-export interface v3ProjectsRelationships {
-  importingUser: v3ProjectsRelashionshipData;
-  org: v3ProjectsRelashionshipData;
-  owner: v3ProjectsRelashionshipData;
-  target: v3ProjectsRelashionshipData;
+export interface RESTProjectsRelationships {
+  importingUser: RESTProjectsRelashionshipData;
+  org: RESTProjectsRelashionshipData;
+  owner: RESTProjectsRelashionshipData;
+  target: RESTProjectsRelashionshipData;
 }
 
-export interface v3ProjectsRelashionshipData {
+export interface RESTProjectsRelashionshipData {
   data: {
     id: string;
     type: string;
   }
   links: unknown;
   meta: unknown;
 }
+
+export interface SnykTargetRESTRelationships {
+  org: {
+    data : {
+      id: string;
+      type: string;
+    }
+    links: unknown;
+    meta: unknown;
+  }
+}
+
+export interface SnykTargetRESTData {
+  attributes: {
+    displayName: string; 
+    isPrivate: boolean;   
+    origin: string;
+    remoteUrl: string | null;
+  }
+  id: string;    
+  relationships: SnykTargetRESTRelationships;
+  type: string;
+}
+
+export interface SnykTargetREST {
+  data: SnykTargetRESTData[];
+  jsonapi: {
+    version: string;
+  }
+  links: unknown;
```

Comment:
Can this `unknown` be replaces with the actual type based on the API docs please?